src/main/java/dev/jbang/cli/Build.java

```diff
@@ -1,12 +1,19 @@
 package dev.jbang.cli;
 
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 @Command(name = "build", description = "Compiles and stores script in the cache.")
 public class Build extends BaseBuildCommand {
 
+	@CommandLine.Option(names = { "-O",
```

Comment:
why capital O ?

Reply:
lowercase is already taken by `--offline` :shrug:

Reply:
Maybe this is a good case for a breaking change? =)
As `-o` is used everywhere for output, a capital O can be an annoyance.

Reply:
> As `-o` is used everywhere for output, a capital O can be an annoyance.

well, if we do things right offline would be used more than output ..I don't think there is really a good choice here. for maven it is `mvn -o` and gradle its `gradle --offline` ...i'm +0 and just keep it as is.